src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql

```diff
@@ -0,0 +1,665 @@
+			        and r.welsh = true
+			        and r.reply_type = 'Digital'
+        group by 	date_trunc('month', h.date_created)) as m
+    on
+    	t.summons_month = m.summons_month
+  	when matched then
+		update set welsh_response_count = m.welsh_response_count
+  	when not matched then
+   		insert values (m.summons_month, m.welsh_response_count);
+
+exception
+	when others then
+    	get stacked diagnostics	v_text_var1 = message_text,
+                            	v_text_var2 = pg_exception_detail,
+                            	v_text_var3 = pg_exception_hint;
+
+   raise notice '%', 'welsh_online_responses failed - error:->' || v_text_var1 || '|' || v_text_var2 || '|' || v_text_var3;
+
+   rollback;
+
+end;
+
+$$;
+
+
+create or replace procedure juror_dashboard.thirdparty_online(no_of_months int)
+language plpgsql as
+
+$$
+/*
+ * Populates juror_dashboard.stats_thirdparty_online
+ * used by the Juror Digital performance dashboard
+ *
+ * Number of third party respones by month summons issued
+ *
+ * Gets date of summons from part_hist
+ *
+ * Replaces the latest n months of summons months as specified by the no_of_months input parameter
+ */
+declare
+
+	v_text_var1 text;
+   	v_text_var2 text;
+   	v_text_var3 text;
+	l_job_type	varchar(50);
+
+begin
+
+	l_job_type := 'refresh_stats_data.thirdparty_online';
+
+    merge into juror_dashboard.stats_thirdparty_online t
+    using (
+        select 		date_trunc('month', h.date_created) summons_month, count(1) thirdparty_response_count
+        from 		juror_mod.juror_response r
+        join 		juror_mod.juror_history h
+        	on 		h.juror_number = r.juror_number
+        where 		h.history_code = 'RSUM'
+        			and h.date_created > current_date - (no_of_months || ' month')::interval  -- exclude jurors summoned more than n months ago
+        			and r.relationship is not null
+        			and r.reply_type = 'Digital'
+        group by 	date_trunc('month', h.date_created)) m
+    on
+    	t.summons_month = m.summons_month
+	when matched then
+		update set thirdparty_response_count = m.thirdparty_response_count
+	when not matched then
+		insert values (m.summons_month,m.thirdparty_response_count);
+
+exception
+	when others then
+		get stacked diagnostics v_text_var1 = message_text,
+        	                    v_text_var2 = pg_exception_detail,
+            	                v_text_var3 = pg_exception_hint;
+
+	raise notice '%', 'thirdparty_online failed - error:->' || v_text_var1 || '|' || v_text_var2 || '|' || v_text_var3;
+
+	rollback;
+
+end;
+
+$$;
+
+
+create or replace procedure juror_dashboard.not_responded(no_of_months int)
+language plpgsql as
+
+$$
+/*
+ * Populates juror_dashboard.stats_not_responded
+ * used by the Juror Digital performance dashboard
+ *
+ * Replaces the latest n months of summons counts as specified by the no_of_months input parameter
+ *
+ * It is recommended to use use no_of_months = 6
```

Comment:
Could we update the comment to remove the duplicated use please?

Reply:
Removed